lib/docker.js

```diff
@@ -451,15 +451,26 @@ function updateDeploy() {
             return startContainer(['-v', opts.dir + ':/opt/service', '-w', '/opt/service'], true);
         }).then(function() {
             // remove unnecessary files
-            return promisedSpawn([
-                'find',
-                'node_modules/',
-                '-regextype',
-                'posix-egrep',
-                '-iregex',
-                '(.*\\.git.*|.*\\.md|.*readme|.*licence)',
-                '-exec', 'rm', '-rf', '{}', ';'
-            ], { capture: true, ignoreErr: true });
+            if (os.type() === 'Darwin') {
```

Comment:
A minor style nit: how about setting an array based on the OS and then doing

```js
return promisedSpawn(findArr, { capture: true, ignoreErr: true });
```

IMHO, it'd make the code more legible.